src/components/tag/tag.visualspec.js

```diff
@@ -0,0 +1,13 @@
+import { percySnapshot } from '@percy/puppeteer';
+import { widths } from '../../../test/percy/widths';
+
+describe('Tag', () => {
+  beforeAll(async () => {
+    await page.goto('http://localhost:3001/tag');
+  });
+
+  it('Tag', async () => {
```

Comment:
Is this „Normal“? Otherwise it is double Tag 😁

Reply:
you mean the naming? What really matters to use is the name we pass to percySnapshot, as that's what shows up on the dashboard there. The Describe and It blocks are only used by jest puppeteer, which assuming the routes were correctly created, will always pass. The passing and failing of the VRT tests happens in Percy and not here 🤷‍♂️

Reply:
Maybe instead of 'Tag', here it would be 'Default'.

Reply:
Yes, that‘s what I meant. Describe is component name and it is default.